### PR TITLE
Added slugs per cubic foot to density

### PR DESCRIPTION
**Pull request by bplett**
I hope I'm doing this right!

### CODE REVIEW

**Comment by anjdreas** [comment]
You need to close the solution, then run [generate-code.bat](https://github.com/anjdreas/UnitsNet/blob/master/generate-code.bat), then edit [DensityTests.cs](https://github.com/anjdreas/UnitsNet/blob/master/UnitsNet.Tests/CustomCode/DensityTests.cs) to implement values for the new test case.

File: UnitsNet/UnitDefinitions/Density.json [hunk 1]
```diff
@@ -110,6 +110,18 @@
           "Abbreviations": [ "t/m³" ]
         }
       ]
+    },
+    {
+      "SingularName": "SlugPerCubicFoot",
+      "PluralName": "SlugsPerCubicFoot",
+      "FromUnitToBaseFunc": "x*515.378818",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/Density.json]: The JSON and conversions look good to me.

**Comment by anjdreas** [comment]
If you missed it, there is a section on https://github.com/anjdreas/UnitsNet#want-to-contribute

**Comment by bplett** [comment]
Thanks!  I knew I had seen that somewhere a long time ago, but couldn’t quickly find it today.  PEBKAC  ☺

Brad.

File: UnitsNet.Tests/CustomCode/DensityTests.cs [hunk 2]
```diff
@@ -68,5 +70,15 @@ public static void DensityTimesKinematicViscosityEqualsDynamicViscosity()
             DynamicViscosity dynamicViscosity = Density.FromKilogramsPerCubicMeter(2) * KinematicViscosity.FromSquareMetersPerSecond(10);
             Assert.AreEqual(dynamicViscosity, DynamicViscosity.FromNewtonSecondsPerMeterSquared(20));
         }
+
+        private static double SlugsPerCubicFootTolerance { get { return 1e-5; } }
+
+        [Test]
```
Comment (anjdreas) [inline UnitsNet.Tests/CustomCode/DensityTests.cs]: This test can be removed, it is already covered by the generated `DensityTests.g.cs`.

**Comment by bplett** [comment]
OK, I think I've completed everything requested.  Do I need to close this pull request and open a new one, or what happens next?

**Comment by anjdreas** [comment]
Next up is for me to release nuget, will do later today.

**Comment by bplett** [comment]
COOL!!!  Thanks a lot!

**Comment by anjdreas** [comment]
Nuget 3.53 is now out

**Pull request merged**